server/src/main/proto/spine/server/entity/entity.proto

```diff
@@ -72,3 +72,13 @@ message LifecycleFlags {
     //
     bool deleted = 2;
 }
+
+// Enumeration of possible entity state validation technical error reasons.
```

Comment:
There are 7 words in this sentence which are not united with anything like prepositions or whatever. This is too complex.

It's better to have two paragraphs (the first one provides the general defintion, the latter adds more details), than have a sentence like this one.

Please rephrase.